### DISCUSSION TITLE
What are the free limitations in pokethost

### DISCUSSION THREAD

**Discussion by Yaseen-Saad** [pockethost/pockethost#395] [Q&A]
Hey guys,
I am new to pockethost and Iam just trying to host my node.js express app "it is a game actually, just like treasure hunt, I made it to the new school generation at my school to help them cope with the school more fast" so I am concerned with how can I run my express app and what are the limitations for free plan as I have like 400 student who are like spamming the website while playing the game.

**Comment by benallfree** [comment]
You will be fine :) 

We are unlimited "Fair Use" which only 4 out of 10,000 devs have ever exceeded.
